app/i18n/de/index.php

```diff
@@ -40,7 +40,7 @@
 		'mark_all_read' => 'Alle als gelesen markieren',
 		'mark_cat_read' => 'Kategorie als gelesen markieren',
 		'mark_feed_read' => 'Feed als gelesen markieren',
-		'mark_selection_unread' => 'Mark selection as unread',	//TODO
+		'mark_selection_unread' => 'Auswahl als gelesen markieren',
```

Comment:
I think you missed an `un(gelesen)`. ;-)

Reply:
Ah chute I missed that
```suggestion
		'mark_selection_unread' => 'Auswahl als ungelesen markieren',
```

Reply:
You'll have to commit the change yourself. I don't have write access to your repo. :-)

Reply:
A thank you, I am new to the whole committing thing :-) I hope that did it...